v0.0.1/admin-schema.json

```diff
@@ -0,0 +1,633 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v0.0.1/admin-schema.json",
+    "title": "Hub administrative settings",
+    "description": "This JSON file provides a schema for modeling hub administrative information.",
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "type": "string",
+            "format": "uri"
```

Comment:
should this be `url`?

Reply:
There is not [url format option in json schema, only uri](https://json-schema.org/understanding-json-schema/reference/string.html#resource-identifiers). But I think that's fine as [URLs are subsets of URIs](https://www.hostinger.com/tutorials/uri-vs-url#:~:text=URI%20identifies%20a%20resource%20and,a%20domain%20name%20and%20port.).